plane/source/docs/qhover-mode.rst

```diff
@@ -121,6 +116,13 @@ Common Problems
    it within an appropriately ventilated enclosure.
 #. Sudden altitude changes caused by light striking the barometer. 
    Assuring sunlight cannot hit the baro will cure this.
+#. QuadPlane slowly descends or climbs until the pilot retakes control in
+   stabilize.  Normally this is caused by not having the throttle stick
+   in the mid position.  This commonly happens when the pilot is
+   switching into an altitude holding mode from a manual flight mode (like QSTABILIZE) on
+   a QuadPlane that does not hover at mid throttle.Usually, it is desired to hover in any mode, at mid-stick on throttle, so that transitions between modes is easily accomplished without throttle position changes. This can be adjusted using the :ref:`Q_M_THST_HOVER<Q_M_THST_HOVER>` parameter, or automatically learned in QHOVER or QLOITER modes by enabling :ref:`Q_M_HOVER_LEARN<Q_M_HOVER_LEARN>`.
+
+.. note:: If :ref:`Q_THROTTLE_EXPO<Q_THROTTLE_EXPO>` = 0 , then :ref:`Q_M_THST_HOVER<Q_M_THST_HOVER>`, whether set manually or learned via Q_M_HOVER_LEARN, is not applied, and the throttle is determined directly from the RC input.
```

Comment:
as this is the QHOVER page this is not true

Reply:
done